etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiOrder.java

```diff
@@ -11,14 +14,17 @@
  */
 public class HapiOrder implements Order<Bundle> {
 
+    private final HapiMessageHelper MESSAGE_HELPER =
```

Comment:
Should this name be in lowercase given it's not static? Was it meant to be static?

Reply:
No. Conventionally, final variables are all uppercase and use underscore. Sonarcloud is just angry and wants us to follow the camel case convention. I think we can leave it as is, thoughts?

Reply:
Sure, if that's the naming convention in Java then we should keep it. Thanks Jorge